CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/commands/NBTUtils.java

```diff
@@ -119,14 +119,14 @@ public static String getAppealingString(String string) {
     }
     
     private static void addNewLine(StringBuilder s, int indent) {
-        s.append("\n\u00A7e├");
+        s.append("\n\u00A7e\u251c");
         for(int j = 0; j < indent; j++) {
             s.append("\u00A7e    ");
         }
     }
     
     private static void addNewLineNoColor(StringBuilder s, int indent) {
-        s.append("\n├");
+        s.append("\n\u251c");
```

Comment:
What does this do?

Reply:
Due to encoding issue on my computer, the character make the my compile work fail. Yeah, it is issue on my side, I shouldn't pushed there.

Reply:
btw, the unicode is `├`. Leaving it don't change anything.